styles.css

```diff
@@ -16,6 +16,7 @@ a {
 }
 
 body {
+  background-color: #e5e5e5;
```

Comment:
- [ ]  Good job so far but you still need some improvement for your project to look awesome , As we are imitate Figama Design and if we compare your design to the figma one are not matching , your design should look the same as the figma design given for this template 😉 
Your Design 👇 
![man1](https://user-images.githubusercontent.com/88894525/192122051-79702802-c1b0-4522-b0ec-ad83a72096eb.png)

Figma Design 👇 

![figmaman](https://user-images.githubusercontent.com/88894525/192122126-f4c85b63-92be-4059-8658-cf6a19a21e23.png)